webdriver-spec.html

```diff
@@ -3237,183 +3237,428 @@ <h2 id=element-displayedness>Element Displayedness</h2>
-                            <li>Let <code>result</code> be the first argument passed to <code>callback</code>.</li>
-                            <li>Set the command's response <a href='#response-status'>status</a> to Success.</li>
-                            <li>Let <code>value</code> be the result of the following algorithm:
-                              <ol>
-                                <li>If <code>result</code> is:
-                                  <ol>
-                                    <li><code>undefined</code> or <code>null</code>, return <code> null</code>.</li>
-                                    <li>a long, boolean, or DOMString, return <code>result</code>.</li>
-                                    <li>a <a href='http://w3c.github.io/dom/#element'>Document <code>element</code></a>, then return the corresponding WebElement for that <a href='http://w3c.github.io/dom/#element'>Document <code>element</code></a>.</li>
-                                    <!-- Should there be a recursion limit? -->
-                                    <li>a <code>Sequence &lt;Node> Nodes</code>, then return the result by recursively applying this algorithm to <code>result</code>. WebDriver implementations SHOULD limit the recursion depth.</li>
-                                    <!-- Should there be a limit? -->
-                                    <li>an object, then return the <code>object</code> created by recursively applying this algorithm to each property in <code>result</code>.</li>
-                                  </ol>
-                                </li>
-                              </ol>
-                            </li>
-                            <li>Set the command's response <a href="#response-value">value</a> to <code>value</code>.</li>
-                          </ol>
-                        </li>
-                        <li>Return the response.</li>
-                      </ol>
-                    </li>
-                  </ol>
-                </section>
-        </section>
-        <section>
-          <h2>Reporting Errors</h2>
-          <p></p>
-        </section>
+            </dd>
+
+            <dt>Otherwise:</dt>
+            <dd>
+              <ol>
+                <li><p>Let <var>result</var> be a
+                    new <code>Object</code> object.</p></li>
+                </ol>
+            </dd>
+          </dl>
+        </li>
+
+        <li><p>For each enumerable own property in <var>value</var>,
+        run the following substeps:</p>
+          <ol>
+            <li><p>Let <var>name</var> be the name of the property.</li>
+
+            <li><p>Let <var>source property value</var> be the result
+                of <a>getting a property</a> named <var>name</var>
+                from <var>value</var>. If doing so causes script to be
+                run, and that script throws an exception,
+                return <a>error</a> with code <a>javascript
+                error</a>.</li>
```

Comment:
"Implementation defined" seems transitive; I don't think that we can call it implementation defined and mandate a specific value. I think we could add a note suggesting that for javascript errors the message and stacktrace properties should match those on the corresponding js object, where available.